testing/directive-tests/square-selection-test.js

```diff
@@ -0,0 +1,51 @@
+(function(){
+    'use strict';
+    describe('Directive test for the square selection functionality', function(){
+        var compile;
+        var rootScope;
+
+
+
+        beforeEach(
+            module(function($provide){
+                $provide.value('tombola.noughtsAndCrossesApp.directive' ,'gameSquare', mocks.directiveMocks, function(value){
```

Comment:
This still needs fixing - if you need help shout. 
$provide.value provides the second argument when the first is requested on the injector.
In your case, it will provide the string 'gameSquare' when 'tombola.noughtsAndCrossesApp.directive' is being asked for....